django/contrib/admin/static/admin/js/inlines.js

```diff
@@ -97,6 +97,7 @@
                     // If a post-delete callback was provided, call it with the deleted form:
                     if (options.removed) {
                         options.removed(row);
+                        $(document).trigger('formset:removed', [row, options.prefix]);
```

Comment:
These events should be triggered regardless of whether the callback is given. The use case here is that a third party plugin can't specify the callback and that is why they need the event to fire.